slice-test/slice-mapper-test/src/test/groovy/com/cognifide/slice/mapper/annotation/IgnorePropertyTest.groovy

```diff
@@ -0,0 +1,28 @@
+package com.cognifide.slice.mapper.annotation
+
+import com.cognifide.slice.test.setup.BaseSetup
+import org.junit.Assert
+
+/**
+ * @author Mariusz Kubiś Date: 14.04.15
+ */
+class IgnorePropertyTest extends BaseSetup {
```

Comment:
It would be nice to see test for Ignoring properties when:
* various mapping strategies are used
* inheritance is used
* field value is set in code.